config/src/test-support/java/org/hyperledger/besu/config/StubGenesisConfigOptions.java

```diff
@@ -262,6 +269,11 @@ public StubGenesisConfigOptions istanbulBlock(final long blockNumber) {
     return this;
   }
 
+  public StubGenesisConfigOptions muirGlacierBlock(final long blockNumber) {
+    muirGlacierBlockNumber = OptionalLong.of(blockNumber);
+    return this;
+  }
```

Comment:
This is unused, does it need to be used in `ReferenceTestProtocolSchedules`?

Reply:
Yep.  Added.